store/tikv/oracle/oracles/pd.go

```diff
@@ -158,15 +178,19 @@ func (o *pdOracle) setLastTS(ts uint64, txnScope string) {
 	}
 }
 
-func (o *pdOracle) getLastTS(txnScope string) (uint64, bool) {
+func (o *pdOracle) getLastTS(txnScope string) (uint64, uint64, bool) {
```

Comment:
I prefer to separate it into 2 functions: getLastTS and getLastArrivalTS.

Reply:
updated.